src/server/rpc/procedures/eclipse-2017/eclipse-2017.js

```diff
@@ -0,0 +1,201 @@
+const Storage = require('../../../storage/storage'),
+    Logger = require('../../../logger'),
+    eclipsePathCenter = require('../../../../../utils/rpc/eclipse-2017/eclipsePathCenter.js'),
+    rpcUtils = require('../utils'),
+    stationUtils = require('./stations.js'),
+    logger = new Logger('netsblox:eclipse'),
+    storage = new Storage(logger);
+
+const STATIONS_COL = 'wuStations',
+    READINGS_COL = 'wuReadings';
+var connection,
+    latestReadings = {};
+
+let eclipsePath = function(){
+    return eclipsePathCenter();
+};
+
+// connect to nb database
+let dbConnect = () => {
+    if (!connection) {
+        connection = storage.connect();
```

Comment:
don't directly call `storage.connect`. This could cause issues with the tests as it will use the production data for the tests. Use the rpc storage object and just create multiple of them